packages/product/package.json

```diff
@@ -42,7 +42,7 @@
     "jest": "^25.5.4",
     "medusa-test-utils": "^1.1.40",
     "pg-god": "^1.0.12",
-    "rimraf": "^5.0.0",
+    "rimraf": "^3.0.2",
```

Comment:
@adrien2p we had to downgrade rimraf in the product module package due to some issues with clashing dependencies. Do you foresee any issues with that?

Reply:
cc @carlos-r-l-rodrigues @riqwan

Reply:
It is used only to delete the folder `/dist` before building. All good.